pkg/resource/db_parameter_group/hooks.go

```diff
@@ -214,3 +252,392 @@ func equalStrings(a, b *string) bool {
 	}
 	return (*a == "" && b == nil) || *a == *b
 }
+
+// syncParameters keeps the resource's parameters in sync
+//
+// RDS does not have a DeleteParameter or DeleteParameterFromParameterGroup API
+// call. Instead, you need to call ResetDBParameterGroup with a list of DB
+// Parameters that you want RDS to reset to a default value.
+func (rm *resourceManager) syncParameters(
```

Comment:
Will we need to same things for `DBClusterParameterGroup`?

Reply:
yes. once I get this done, will work on that